### ISSUE TITLE
Domain Admin can't configure Limits for a regular User Account in the same domain

### ISSUE THREAD

**Issue by msinhore** [apache/cloudstack#5676]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.16.0
~~~

##### SUMMARY

When a domain admin try to set limits for a regular user account in the same domain level, the tab Configure Limits is not shown.  

##### STEPS TO REPRODUCE

~~~
- Create a domain
- Add a domain admin account
- Do login with domain admin account
- Create a regular user account
- Click on regular user account to configure limits
~~~

##### EXPECTED RESULTS

~~~
As domain admin could configure the limits of a regular user account.
~~~

##### ACTUAL RESULTS

~~~
The Configure Limits tab is not shown.
~~~

**Comment by weizhouapache** [comment]
@msinhore 
thanks for creating the issue.
I would regard it as a feature request than bug report.

**Comment by msinhore** [comment]
@weizhouapache 
In previous relases it was working. I think something broke in 4.16.

**Comment by weizhouapache** [comment]
> @weizhouapache In previous relases it was working. I think something broke in 4.16.

@msinhore got it. if so, it is a bug.

**Comment by DaanHoogland** [comment]
@msinhore can you explain what you expect? I tried both 4.15 and 4.14 and cannot see any limit configuration tab on accounts in either.

**Comment by shwstppr** [comment]
@msinhore is this related to this view for editing account resource count?
![Screenshot from 2022-01-13 15-53-35](https://user-images.githubusercontent.com/153340/149312429-dc7f97f3-bb36-4ef6-a20d-715441289157.png)

**Comment by shwstppr** [comment]
@msinhore @DaanHoogland created PR #5858 to restore Configure limits tab in the UI for domain-admins. Please see if this fixes the issue.
Need to check if it was explicitly for some reason.

**Comment by DaanHoogland** [comment]
fixed in #5858
